.github/workflows/sanitizer.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: configure msan env
       if: matrix.sanitizer == 'msan'
       run: |
-        echo "EXTRA_FLAGS=-g -O2 -fno-omit-frame-pointer -fsanitize=memory -fsanitize-memory-track-origins" >> $GITHUB_ENV
+        echo "EXTRA_FLAGS=-g -O2 -fno-omit-frame-pointer -fsanitize=memory -fsanitize-memory-track-origins -fsanitize-address-use-odr-indicator" >> $GITHUB_ENV
```

Comment:
It should be for `if: matrix.sanitizer == 'asan'` block, though

Reply:
ugh. i forgot what i was doing over the weekend. sorry.

Reply:
this is the default for non-windows so i can drop it anyway